src/pages/Schedule.tsx

```diff
@@ -1,7 +1,320 @@
-import React from "react";
+import {
+  Box,
+  Button,
+  Container,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Radio,
+  RadioGroup,
+  Select,
+  Slider,
+  SliderFilledTrack,
+  SliderThumb,
+  SliderTrack,
+  Stack,
+  Textarea,
+  Tooltip,
+  Table,
+  Thead,
+  Tbody,
+  Tfoot,
+  Tr,
+  Th,
+  Td,
+  TableCaption,
+  TableContainer,
+} from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Link } from "react-router-dom";
+import { gql, useMutation } from "@apollo/client";
+import { useLoginStore } from "../stores/login";
+
+// These schemas will probably change later, all just example data
+const SUBMIT = gql`
+  mutation Login($username: STring!, $password: STring!) {
```

Comment:
```suggestion
  mutation Login($username: String!, $password: String!) {
```
Probably should have restricted your find-and-replace to just the JSX :stuck_out_tongue: